src/main/java/com/google/netpcapanalysis/dao/GeolocationDaoImpl.java

```diff
@@ -0,0 +1,39 @@
+package com.google.netpcapanalysis.dao;
+
+import com.google.netpcapanalysis.interfaces.dao.GeolocationDao;
+import com.maxmind.geoip2.DatabaseReader;
+import com.maxmind.geoip2.model.CityResponse;
+import com.maxmind.geoip2.record.Country;
+import java.io.File;
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.URL;
+
+public class GeolocationDaoImpl implements GeolocationDao {
+
+  private File database;
+  private DatabaseReader reader;
+
+  public GeolocationDaoImpl() {
+    try {
+      URL geoDBUrl = GeolocationDaoImpl.class.getClassLoader().getResource("GeoLite2-City.mmdb");
```

Comment:
Declare the resource file name as a final static variable. Ideally we could read this from a properties file but that is future.

Reply:
Fixed